### ISSUE TITLE
False positive: Use colon rule on constructor with dataset parameter

### ISSUE THREAD

**Issue by cverbiest** [Riverside-Software/sonar-openedge#755]
I get _Use colon instead of period to start a code block (rssw-oe-main:eu.rssw.antlr.proparse.checks.ColonForBlock)`_

on 
```
    constructor public CceDesignValidation (dataset for dsBusinessEntity bind).

    end constructor.
```

**Comment by movedoa** [comment]
You are using a period instead of a colon here.
Am i missing something?

**Comment by cverbiest** [comment]
@movedoa No you're not, I was
